app/src/main/java/com/shvants/runninglife/MainActivity.java

```diff
@@ -1,13 +1,35 @@
 package com.shvants.runninglife;
 
+import android.content.ComponentName;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.ServiceConnection;
 import android.os.Bundle;
+import android.os.IBinder;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
-import android.widget.Button;
 
 
 public class MainActivity extends AppCompatActivity {
 
+    public static final String RESULT = "RESULT";
```

Comment:
There is already such field in CounterBroadcastReceiver

Reply:
moved this constant in 'Const' class https://github.com/AliakseiShvants/RunningLife/compare/01842e8f4dd29c265bdc458cc8a8a71b6eae8310...58e65bc2b5d572145ec44afbcbc683ee2b932035